units/host_monitoring_idle_termination.go

```diff
@@ -24,7 +24,7 @@ const (
 	idleWaitingForAgentCutoff = 10 * time.Minute
 
 	// outdatedIdleTimeCutoff is the amount of time we wait for an outdated idle host to be marked idle.
-	outdatedIdleTimeCutoff = time.Minute
+	outdatedIdleTimeCutoff = 5 * time.Second
```

Comment:
Would it make sense to make this 0 (and change the `>=`s to `>`?

Reply:
Like just don't wait any amount of time? That would make sense if we want to be super aggressive about the termination, is that what you're thinking?

Reply:
I guess I'm not sure what the difference really is between 0 and 5 seconds. Do you have a sense how fast nextTask is? Like will 5 seconds actually allow nextTask to run?